frontend/src/components/Menu.js

```diff
@@ -82,9 +82,9 @@ const Menu1 = () => {
               <li>
                 <a href="/#nav5">Submit</a>
               </li>
-              <li>
+              /*<li>
                 <a href="/logreg">Register/Login</a>
-              </li>
+              </li>*/
```

Comment:
This is a superficial fix, the component and its modals need to be removed from the application as well.